poc/frost.sage

```diff
@@ -196,7 +197,7 @@ def trusted_dealer_keygen(G, n, t):
 # Configure the setting
 NUM_SIGNERS = 3
 THRESHOLD_LIMIT = 2
-G = GroupRistretto255()
+G = GroupEd25519() # GroupRistretto255() # GroupP256()
```

Comment:
👀

Reply:
I'll tidy this up before I promote this PR for real review.